mne/io/proj.py

```diff
@@ -100,6 +100,13 @@ def add_proj(self, projs, remove_existing=False):
 
         return self
 
+    def add_eeg_ref(self):
+        """Add an average EEG reference projector if one does not exist
+        """
+        if _needs_eeg_average_ref_proj(self.info):
+            eeg_proj = make_eeg_average_ref_proj(self.info, activate=True)
```

Comment:
Ahh -- I think this actually needs to be `False` now that I think about it. IIRC `activate` means set the `proj['active'] = activate`, and `proj['active']` says that the projector has been applied to the data already. @agramfort is that right?